addons/website/models/res_config.py

```diff
@@ -16,32 +16,59 @@ def _default_website(self):
     website_name = fields.Char('Website Name', related='website_id.name')
 
     language_ids = fields.Many2many(related='website_id.language_ids', relation='res.lang')
-    default_lang_id = fields.Many2one(related='website_id.default_lang_id', relation='res.lang', required=True)
+    number_of_languages = fields.Integer(string='Number of languages', compute='_compute_number_of_languages', readonly=True)
+    default_lang_id = fields.Many2one(string='Default', related='website_id.default_lang_id', relation='res.lang', required=True)
     default_lang_code = fields.Char('Default language code', related='website_id.default_lang_code')
-    google_analytics_key = fields.Char('Google Analytics Key', related='website_id.google_analytics_key')
-
-    social_twitter = fields.Char(related='website_id.social_twitter')
-    social_facebook = fields.Char(related='website_id.social_facebook')
-    social_github = fields.Char(related='website_id.social_github')
-    social_linkedin = fields.Char(related='website_id.social_linkedin')
-    social_youtube = fields.Char(related='website_id.social_youtube')
-    social_googleplus = fields.Char(related='website_id.social_googleplus')
-    cdn_activated = fields.Boolean('Use a Content Delivery Network (CDN)', related='website_id.cdn_activated')
+    google_analytics_key = fields.Char('Analytics Key', related='website_id.google_analytics_key')
```

Comment:
For those fields, same as above: I would keep Google _* in string, and maybe use a different label in views.